beta/src/pages/learn/javascript-in-jsx-with-curly-braces.md

```diff
@@ -67,11 +66,11 @@ export default function Avatar() {
 
 </Sandpack>
 
-Notice the difference between `className="avatar"`, which specifies an `"avatar"` CSS class name that makes the image round, and `src={avatar}` that reads the value of the JavaScript variable called `avatar`. That's because curly braces let you work with JavaScript right there in your markup!
+क्लासनाम = "अवतार" के बीच अंतर पर ध्यान दें, जो एक "अवतार" CSS वर्ग का नाम निर्दिष्ट करता है जो छवि को गोल बनाता है, और src={अवतार} जो अवतार नामक JavaScript चर के मूल्य को पढ़ता है। ऐसा इसलिए है क्योंकि कर्ली ब्रेसेस आपको अपने मार्कअप में वहीं JavaScript के साथ काम करने देते हैं!
```

Comment:
can you revisit this sentence, the sentence seems weird and does not makes sense